packages/schemas/tables/hooks.sql

```diff
@@ -1,13 +1,16 @@
 create table hooks (
-    tenant_id varchar(21) not null
-      references tenants (id) on update cascade on delete cascade,
-    id varchar(21) not null,
-    event varchar(128) /* @use HookEvent */ not null,
-    config jsonb /* @use HookConfig */ not null,
-    created_at timestamptz not null default(now()),
-    primary key (id)
-  );
+  tenant_id varchar(21) not null
+    references tenants (id) on update cascade on delete cascade,
+  id varchar(21) not null,
+  name varchar(256),
+  event varchar(128) /* @use HookEvent */, /* deprecated */
+  events jsonb /* @use HookEvents */,
+  config jsonb /* @use HookConfig */ not null,
+  enabled boolean not null default true,
```

Comment:
should the default value be true? Was thinking the default value is false and users can manually set it to true after testing.

Reply:
This is by designed.

Reply:
why it's by design?

Reply:
Aligned with the designer.